docs/source/java/memory.rst

```diff
@@ -133,7 +133,7 @@ Development Guidelines
 Applications should generally:
 
 * Use the BufferAllocator interface in APIs instead of RootAllocator.
-* Create one RootAllocator at the start of the program.
+* Create one RootAllocator at the start of the program (in case `Singleton Allocator`_ is needed, consider using it).
```

Comment:
I would generally recommend against using it...

Reply:
Deleted